README.md

```diff
@@ -20,12 +20,14 @@ yarn storybook
 
 ### Mobile
 
-After running `yarn`, you also need to link assets using:
```

Comment:
Is this information somewhere else? Or is linking not necessary at all?

Reply:
Linking for https://github.com/kiwicom/universal-components/blob/master/package.json#L96-L100.